packages/twenty-front/src/modules/object-record/record-filter/utils/compute-record-gql-operation-filter/for-composite-field/computeGqlOperationFilterForEmails.ts

```diff
@@ -0,0 +1,153 @@
+import { FieldMetadataItem } from '@/object-metadata/types/FieldMetadataItem';
+import {
+  EmailsFilter,
+  RecordGqlOperationFilter,
+} from '@/object-record/graphql/types/RecordGqlOperationFilter';
+
+import { RecordFilter } from '@/object-record/record-filter/types/RecordFilter';
+import { RecordFilterOperand } from '@/object-record/record-filter/types/RecordFilterOperand';
+import { CompositeFieldSubFieldName } from '@/settings/data-model/types/CompositeFieldSubFieldName';
+import { isNonEmptyString } from '@sniptt/guards';
+
+export const computeGqlOperationFilterForEmails = ({
```

Comment:
I might not understand the code but it seems that this is computeContainsGqlOperationFilterForEmails

Reply:
That's not what it is intended, those utils should just act as the breaking down of each switch case on the field type in the big `turnRecordFilterIntoRecordGqlOperationFilter`, not as a specific implementation of a particular operand.

The operands can vary between the different field types and it can seem like it is about contains operand only because we have not finished yet for all the types, some types have is / is not.

Reply:
I'm considering putting back emptiness operands in this util, as they also differ for each field type and it may not be a good idea to split the logic between `getEmptyRecordGqlOperationFilter` and `turnRecordFilterIntoRecordGqlOperationFilter`, refactoring every case into a separate util will probably reveal the right solution.